tests/test_petab.py

```diff
@@ -5,6 +5,7 @@
 import os
 import libsbml
 import numpy as np
+import pickle
```

Comment:
Add as test requirement to setup.py?

Reply:
it's a python library installed by default, but I'll add it though